workshops/templates/base_without_navbar.html

```diff
@@ -7,12 +7,12 @@
     <link rel="stylesheet" href="{% static 'css/amy.css' %}" />
     <link rel="stylesheet" href="{% static 'css/bootstrap-datepicker3.min.css' %}" />
     <link rel="stylesheet" href="https://code.jquery.com/ui/1.11.4/themes/smoothness/jquery-ui.css" />
+    {% block extrastyle %}{% endblock %}
```

Comment:
@pbanaszkiewicz It turns out we don't need to `include_ui_theme` at all since a jQuery UI theme is included in our base template; I just removed the tag from all templates, so this should be resolved.